python/tvm/auto_scheduler/measure.py

```diff
@@ -628,6 +636,8 @@ def local_build_worker(args):
         build_func = tar.tar
     elif build_func == "ndk":
         build_func = ndk.create_shared
+    elif build_func == "custom":
```

Comment:
if so, we could use BuildFunc.build_func for all conditions

Reply:
I change all conditions to BuildFunc